WordPress/src/main/java/org/wordpress/android/ui/ActivityLauncher.java

```diff
@@ -416,6 +416,11 @@ public static void viewHelpAndSupport(@NonNull Context context, @NonNull Origin
         context.startActivity(HelpActivity.createIntent(context, origin, selectedSite, extraSupportTags));
     }
 
+    public static void viewZendeskTickets(@NonNull Context context,
+                                          @Nullable SiteModel selectedSite, @Nullable List<String> extraSupportTags) {
```

Comment:
`extraSupportTags` are not necessary for push notifications. If that ever changes we can then pass it, no need to pass `null` to it right now.

Reply:
addressed in 76d8cff5be